### ISSUE TITLE
bugs

### ISSUE THREAD

**Issue by jangjiun** [robbebluecp/tf2-yolov4#7]
logits and labels must have the same shape ((None, None, None, 3, 2) vs (None, 13, 13, 3, 20))，搞不懂我把class都改了，这个20是从哪来的？
